stream-buffers/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for stream-buffers 3.0
+// Project: https://github.com/samcday/node-stream-buffer#readme
+// Definitions by: Jason Dent <https://github.com/Jason3S>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+import * as stream from 'stream';
+
+export interface WritableStreamBufferOptions extends stream.WritableOptions {
+    initialSize?: number;
+    incrementAmount?: number;
+}
+
+export declare class WritableStreamBuffer extends stream.Writable {
+    constructor(options?: WritableStreamBufferOptions);
+    size(): number;
+    maxSize(): number;
+    getContents(length?: number): any | false;
```

Comment:
Good point.  I didn't realize the order mattered.
It is more of a hint that it will return false when the contents are empty.
I am happy to leave it with just `any` or reverse the order.